cdk/cdk/cdk_stack.py

```diff
@@ -4,26 +4,37 @@
 import aws_cdk.aws_stepfunctions as stepfunctions
 import aws_cdk.aws_events as events
 import aws_cdk.aws_events_targets as targets
+from aws_cdk import aws_secretsmanager as secretsmanager
 from aws_cdk import aws_lambda
 from aws_cdk import aws_stepfunctions_tasks as tasks
 from aws_cdk import aws_ec2 as ec2
 from aws_cdk import aws_sqs as sqs
-from aws_cdk import aws_s3
+from aws_cdk import aws_s3 as s3
+from aws_cdk.aws_lambda_event_sources import SqsEventSource
 
+SECRET_NAME = os.environ["SECRET_NAME"]
 
 class CdkStack(core.Stack):
     def __init__(self, scope: core.Construct, construct_id: str, **kwargs) -> None:
         super().__init__(scope, construct_id, **kwargs)
+        stack_name = construct_id
+
         collection = "OMDOAO3e"
         version = "003"
 
         bucket = "climatedashboard-data"
         prefix = "OMSO2PCA/"
```

Comment:
presumably we won't need these variables when we remove the S3 trigger from the HLS workflow right?